.github/workflows/ci_mocked_api.yml

```
@@ -88,6 +88,10 @@ jobs:
         run: poetry install
 
       - name: Run pytest
+        env:
+            SNAKEMAKE_AWS_BATCH_REGION: "${{ secrets.SNAKEMAKE_AWS_BATCH_REGION }}"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)